src/configurations/cancercomplexity/resources.ts

```diff
@@ -4,4 +4,4 @@ export const filesSql =
 export const grantsSql = `SELECT * FROM syn21918972`
 export const projectsSql = `SELECT * FROM syn21868602`
 export const publicationSql = 'SELECT * FROM syn21868591'
-export const toolsSql = 'SELECT * FROM syn21930566'
+export const toolsSql = 'SELECT * FROM syn26127427'
```

Comment:
Need to make this ['Open Data'](https://sagebionetworks.jira.com/wiki/spaces/PLFM/pages/792231937/Configuring+Tables+to+Allow+Anonymous+View+Access) before we can merge this change to production

Reply:
Thanks!  Just [filed a JIRA](https://sagebionetworks.jira.com/browse/PLFM-6949) for Xa to review and approve.